app/src/main/java/com/nextcloud/talk/utils/NotificationUtils.kt

```diff
@@ -61,6 +61,10 @@ object NotificationUtils {
 
     const val NOTIFICATION_CHANNEL_MESSAGES_V4 = "NOTIFICATION_CHANNEL_MESSAGES_V4"
     const val NOTIFICATION_CHANNEL_CALLS_V4 = "NOTIFICATION_CHANNEL_CALLS_V4"
+    const val NOTIFICATION_CHANNEL_UPLOADS = "NOTIFICATION_CHANNEL_UPLOADS"
+
+    private const val IMPORTANCE_HIGH = 4 // eq. NotificationManager.IMPORTANCE_HIGH (Build.VERSION_CODES.N)
+    private const val IMPORTANCE_LOW = 2 // eq. NotificationManager.IMPORTANCE_LOW (Build.VERSION_CODES.N)
```

Comment:
important: This is not a good idea as the value in NotificationManager may change in new versions. Instead set a `@TargetApi` in the calling methods, or surround the calls with a version code check